package/wpe/wpeframework-plugins/Config.in

```diff
@@ -603,6 +603,13 @@ config BR2_PACKAGE_WPEFRAMEWORK_SYSTEMCOMMANDS
     help
       SystemCommands Plugin
 
+config BR2_PACKAGE_WPEFRAMEWORK_SECURESHELLSERVER
+    select BR2_PACKAGE_WPEFRAMEWORK_PLUGINS
```

Comment:
I dont think this is needed, you're already in the wpeframework-plugins config right?

Reply:
Yes, removed.